CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributes
+
+To create an extension or run Nango locally: [check our documentation](https://docs.nango.dev/contribute)
+
+## Develop locally
+
+To develop on the platform locally follow those steps:
+
+```sh
+git clone https://github.com/NangoHQ/nango.git
+```
+
+Install the project
+
+```sh
+npm run install:all
+```
+
+Set your envs
+
+```sh
+cp .env.example .env
+```
+
+Launch the services
```

Comment:
it is more launching datastores/queues that services. no?

Reply:
yeah I didn't know how to name that general category, happy to put "datastores/queues"

Reply:
`Launch datastores` is probably good enough. what do you think?